### PR TITLE
[8.0][IMP][base_export_manager] Improve UX

### PR DESCRIPTION
**Pull request by yajo**
- Improve user instructions in README.
- Require some required fields.
- Allow to select models from a list.
- Allow to select up to 3 fields from dynamic lists.
- Improve translations.
- More tests.
- Translate column labels.

Some methods have been renamed, so version tag is raised to 8.0.2.0.0.

### CODE REVIEW

File: base_export_manager/README.rst [hunk 1]
```diff
@@ -43,6 +67,8 @@ Contributors
 
 * Antonio Espinosa <antonioea@antiun.com>
 * Javier Iniesta <javieria@antiun.com>
+* Rafael Blasco <rafabn@antiun.com>
+* Jairo Llopis <yajo.sk8@gmail.com>
 
 Maintainer
 ----------
```
Comment (elicoidal) [inline base_export_manager/README.rst:74]: https

File: base_export_manager/__openerp__.py [hunk 2]
```diff
@@ -23,11 +23,12 @@
 {
```
Comment (elicoidal) [inline base_export_manager/__openerp__.py:5]: Please take the opportunity to simplify headers.

**Comment by rafaelbn** [comment]
Travis fails

File: base_export_manager/README.rst (hunk 1 above)
Comment (yajo) [inline base_export_manager/README.rst:74]: Fixed.

File: base_export_manager/__openerp__.py (hunk 2 above)
Comment (yajo) [inline base_export_manager/__openerp__.py:5]: Fixed.

File: base_export_manager/models/ir_exports.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    def _compute_model_id(self):
+        """Get the model from the resource."""
+        for s in self:
+            s.model_id = self._get_model_id(s.resource)
+
+    @api.multi
+    def _inverse_model_id(self):
+        """Get the resource from the model."""
+        for s in self:
+            s.resource = s.model_id.model
```
Comment (yajo) [inline base_export_manager/models/ir_exports.py:38]: You can see this method is not being called in the runbot when you select a `model_id`. I'd thank help on why.

File: base_export_manager/models/ir_exports_line.py [hunk 4]
```diff
@@ -30,28 +11,119 @@ class IrExportsLine(models.Model):
...
+        "Second field",
+        domain="[('model_id', '=', model2_id)]")
+    field3_id = fields.Many2one(
+        "ir.model.fields",
+        "Third field",
+        domain="[('model_id', '=', model3_id)]")
+    model1_id = fields.Many2one(
+        string="First model",
+        readonly=True,
+        related="export_id.model_id")
```
Comment (yajo) [inline base_export_manager/models/ir_exports_line.py:38]: This field is not working in runbot. It is always false, no matter the value of `export_id.model_id`. I'd thank help on that too.

**Comment by rafaelbn** [comment]
Maybe @JavierIniesta could help

**Comment by rafaelbn** [comment]
Rebuilding for testing

cc @antespi @pedrobaeza

**Comment by moylop260** [comment]
👍 
Please, Could you check the rst syntax error of readme https://travis-ci.org/OCA/server-tools/jobs/99268857?

**Comment by rafaelbn** [comment]
@Yajo  Tested. I'm not able to create or edit an export profile. Could you please check it?

**Comment by yajo** [comment]
@Tecnativa

**Comment by rafaelbn** [comment]
Please @Yajo review this PR, thanks!

**Comment by yajo** [comment]
All should be working not, including ability to reorder fields.

**Comment by rafaelbn** [comment]
Thank-you, perfect! :+1: :smile: :rocket: tested in runbot! After 7 months waiting for this...

Please @moylop260 could you review here again?

cc @lasley done! :wink:

**Comment by moylop260** [comment]
I'm agree yet

**Comment by pedrobaeza** [comment]
Tried on runbot :+1:

**Comment by rafaelbn** [comment]
@Yajo travis is failing, please take a look. When travis will be green this PR is ready to be merged

**Comment by rafaelbn** [comment]
@Yajo travis fails again

**Comment by yajo** [comment]
It seems [the guilty module is fetchmail_attach_from_folder](https://travis-ci.org/OCA/server-tools/jobs/147705061#L1765-L1771).

**Comment by pedrobaeza** [comment]
Please fix it (here or in other PR and rebase this one)

**Comment by lasley** [comment]
Code + functional 👍  

Can confirm, Travis errors are all unrelated and were on 8.0 a few iterations ago. Seems if we rebase on top of 8.0 now though, we will have a [Lint error](https://travis-ci.org/OCA/server-tools/builds/147159732) to replace it 😆

**Comment by pedrobaeza** [comment]
OK, I'm merging this now then and I'll commit directly the fix for the linter.

**Pull request merged**